packages/sui-pde/src/components/experiment.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types'
+import useExperiment from '../hooks/useExperiment'
+
+export default function Experiment({
+  experimentName,
+  attributes,
+  trackExperimentViewed,
+  queryString,
+  children
+}) {
+  const value = useExperiment({
+    experimentName,
+    attributes,
+    trackExperimentViewed,
+    queryString
+  })
+
+  return children(value)
```

Comment:
do you think this would make this part easier to read and grasp?

```suggestion
  const variation = useExperiment({
    experimentName,
    attributes,
    trackExperimentViewed,
    queryString
  })

  return children({variation})
```

Reply:
Yep! thanks @jordevo